packages/camera/camera/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 0.9.4+22
 
+* Fixes preview orientation when pausing preview with locked orientation.
```

Comment:
You lost the version change in a merge; that will need to be fixed before CI will pass.

Reply:
Thanks for noting, this is now fixed.